RNTester/Podfile.lock

```diff
@@ -249,4 +249,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d10549a91f1ecde1496088874e41348b2cd79f9f
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
your pod version is older?

Reply:
Yeah that's odd. I guess I could revert this portion, but not sure what the actual badness is